src/widgets/Tabs/VaccineSiteSelect.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable react/require-default-props */
+/* eslint-disable react/forbid-prop-types */
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2021
+ */
+
+import React, { useState } from 'react';
+
+import { View } from 'react-native';
+
+import { batch, connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { FlexRow } from '../FlexRow';
+import { FlexView } from '../FlexView';
+
+import globalStyles from '../../globalStyles/index';
+import { buttonStrings } from '../../localization/index';
+import { JSONForm } from '../JSONForm/JSONForm';
+import { selectSiteSchemas } from '../../selectors/formSchema';
+
+import { PageButton } from '../PageButton';
+import { PageButtonWithOnePress } from '../PageButtonWithOnePress';
+import { VaccinePrescriptionActions } from '../../actions/Entities/index';
+import { WizardActions } from '../../actions/WizardActions';
+
+const { pageTopViewContainer } = globalStyles;
+
+const VaccineSiteSelectComponent = ({ onCancel, onComplete, siteSchema }) => {
+  const [{ formData, isValid }, setForm] = useState({ formData: null, isValid: false });
+  return (
+    <FlexView style={pageTopViewContainer}>
+      <JSONForm
```

Comment:
I wrapped this in `Paper` to get the same header style.. though that will require the JSON form to lose it's paper-ness in order to look right. Either that or pass in a header style to the JSON form.. either way, it would be nice to update the header style.

Reply:
Tried the `Paper` method...Removed title from the JSON Schema and made the JSON Form Paperisation dependant on the presence of a title 
83f21d0
76bc1a3

```
{
  "type": "object",
  "required": [],
  "properties": {
    "administeringLocation": {
      "type": "string",
      "title": "Administering Location"
    }
  }
}
```

![image](https://user-images.githubusercontent.com/65875762/124414721-59c08c00-dda7-11eb-9bb4-5de7fc76d47c.png)